deltachat-ios/Controller/EditSettingsController.swift

```diff
@@ -46,6 +46,8 @@ class EditSettingsController: UITableViewController, MediaPickerDelegate {
     private lazy var nameCell: TextFieldCell = {
         let cell = TextFieldCell(description: String.localized("pref_your_name"), placeholder: String.localized("pref_your_name"))
         cell.setText(text: dcContext.displayname)
+        cell.textFieldDelegate = self
+        cell.textField.returnKeyType = .continue
```

Comment:
It shall indicate that cou need to continue with the workflow (instead of next, which indicates move to next text input), which is to add an avatar in this case. 
I read some discussions about 'next' vs. 'continue' where this semantics for continue was outlined like described above. Also the 'continue' button might be seen more actionable than the default return. Since the button hides the keyboard which helps for continuing the current task it might be nice to have that feature a little bit more user facing than it would be with the default return button.
Of course these slight UX changes are also very subjective - so if you think the default button is still better, I'll revert that change.

Reply:
do you have a link about these discussions? i did not found much. however, using the "normal" ios ui as a reference, i have not seen "continue" many times, so, if in doubt, i would keep "return", indeed.